plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -152,7 +164,7 @@ export const CatalogPage: FC<{}> = () => {
     (rowData: Entity) => {
       const isStarred = isStarredEntity(rowData);
       return {
-        icon: isStarred ? Star : StarOutline,
+        icon: isStarred ? YellowStar : StarOutline,
```

Comment:
Yeah this really begs to become a separate standalone component soon :) that encapsulates the checking, styling and state changing etc

Reply:
Feel free to create a [good-first-issue] ticket for it! 💯